### PR TITLE
Rename "host.port" attribute to "net.host.port"

### LINKED ISSUE #240: Rename "host.port" attribute to "net.host.port"
According to Semantic Conventions https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/semantic_conventions/http-metrics.md
attribute "net.host.port" should be used instead of "host.port"

### PR DESCRIPTION
**Pull request by malafeev**
# Description

According to Semantic Conventions https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/semantic_conventions/http-metrics.md
attribute "net.host.port" should be used instead of "host.port"

Fixes #240

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] tox -e test-instrumentation-wsgi
- [x] tox -e test-instrumentation-pyramid
- [x] tox -e test-instrumentation-flask
- [x] tox -e test-instrumentation-falcon
- [x] tox -e test-instrumentation-asgi 

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/master/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [x] Followed the style guidelines of this project
- [x] Changelogs have been updated
- [ ] Unit tests have been added
- [x] Documentation has been updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -307,6 +309,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - `opentelemetry-ext-http-requests` Rename package to opentelemetry-ext-requests
   ([#619](https://github.com/open-telemetry/opentelemetry-python/pull/619))
+- - `opentelemetry-ext-http-requests` Rename package to opentelemetry-ext-requests
```
Comment (codeboten) [inline CHANGELOG.md:312]: this looks like a copy and paste typo?

**Review by codeboten — CHANGES_REQUESTED** [review]
Minor change to the changelog, otherwise the PR looks great!

File: CHANGELOG.md (hunk 1 above)
Comment (malafeev) [reply CHANGELOG.md]: fixed

**Review by codeboten — APPROVED** [review]
Thanks!

**Review by lzchen — APPROVED** [review]

**Review by ocelotl — APPROVED** [review]
Thanks, @malafeev :+1:

**Comment by malafeev** [comment]
can anybody merge this PR?

**Pull request merged**